packages/owl/owl.0.3.7/opam

```diff
@@ -18,13 +18,16 @@ build-test: [
 
 depends: [
   "alcotest" {test}
+  "base"
   "base-bigarray"
   "conf-openblas" {>= "0.2.0"}
+  "configurator"
```

Comment:
This is likely a build dependency btw.

Reply:
good point, thanks!

Reply:
`configurator`, `base` and `stdio` seems to be used only in `src/owl/config/configure.ml`, I think they are all build dependencies